tests/routes/__init__.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict
+
+generic_dataset_metadata = {
+    "metadata": {
+        "title": "string",
+        "subtitle": "string",
+        "function": "string",
+        "resolution": "string",
+        "geographic_coverage": "string",
+        "source": "string",
+        "update_frequency": "string",
+        "cautions": "string",
+        "license": "string",
+        "overview": "string",
+        "citation": "string",
+        "tags": ["string"],
+        "data_language": "string",
+        "key_restrictions": "string",
+        "scale": "string",
+        "added_date": "2020-06-25",
+        "why_added": "string",
+        "other": "string",
+        "learn_more": "string",
+    }
+}
+
+
+def create_version(test_client, dataset, version) -> None:
+    # Create dataset and version
+    dataset_resp = test_client.put(f"/meta/{dataset}", json=generic_dataset_metadata)
+    assert dataset_resp.json()["status"] == "success"
```

Comment:
To stick to DRY I would make this its own function as well

Reply:
Done.